valhalla/worker.h

```diff
@@ -74,6 +74,12 @@ void ParseApi(const prime_server::http_request_t& http_request, Api& api);
 #endif
 
 std::string serialize_error(const valhalla_exception_t& exception, Api& options);
+
+// lambda function to add warnings to proto info object
+const auto add_warning = [](Api& api, const std::string& warning_text) {
```

Comment:
we should convert this to a proper function rather than a lambda

Reply:
Okay. I'll update that in the next commit.